bin/bookkeeper

```diff
@@ -166,6 +166,17 @@ OPTS="$OPTS -Dlog4j.configurationFile=`basename $BOOKIE_LOG_CONF`"
 # Allow Netty to use reflection access
 OPTS="$OPTS -Dio.netty.tryReflectionSetAccessible=true"
 
+IS_JAVA_8=`java -version 2>&1 |grep version|grep '"1\.8'`
```

Comment:
What about jdk 11.0.1.8?
Will this check fail?

Reply:
In BK we have another way to detect modern jdks with modules

Reply:
I understand better now, you added a double quote in the beginning so it should match only "1.8